server/build.gradle

```diff
@@ -32,15 +32,15 @@ dependencies {
   implementation 'io.javalin:javalin:3.12.0'
 
   // Mongo DB Driver for Java
-  implementation 'org.mongodb:mongodb-driver:3.12.7'
+  implementation 'org.mongodb:mongo-java-driver:3.12.7'
```

Comment:
These two are not equivalent. Also these particular packages haven't continued through to Mongo 4.

Reply:
Good catch about the Mongo 4, but that's a whole 'nother set up updates, and looks like it's a non-trivial set of changes that we'd have to make. I'll create an issue about it, but I'm not sure it's happening right now.

Reply:
We are already using Mongo 4 so this is probably necessary.